### PR TITLE
(DOCSP-26119) Add sync client reconnect docs [Flutter]

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-26119
Staged changes: [Manage a Sync Session](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/DOCSP-26119/sdk/flutter/sync/manage-sync-session/#manually-reconnect-all-sync-sessions)

The work described in the ticket links to four other RDART tickets. Three of those weren't implemented. The only sync client method to document is `App.reconnect()`.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-app-services work, if any

### Release Notes

- **Flutter SDK**
  - Sync/Manage A Sync Session: Add documentation for `App.reconnect()`.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/flutter/sync/manage-sync-session.txt [hunk 1]
```diff
@@ -124,3 +132,28 @@ You can access the current and previous ``ConnectionState`` from ``ConnectionSta
 
 .. literalinclude:: /examples/generated/flutter/manage_sync_session_test.snippet.monitor-network-connection.dart
    :language: dart
+
+Manually Reconnect All Sync Sessions
+------------------------------------
+
+Realm automatically detects when a device regains connectivity after being
```
Comment (dacharyc) [inline source/sdk/flutter/sync/manage-sync-session.txt:139]: Just a Q - do we want to scrub Realm naming per our new naming guidelines on new content that we add to pages going forward? I scrubbed it from the geospatial page I added. I know we need to do a bigger effort to address it retroactively at some point, but if we don't _add_ more instances of Realm naming, we save ourselves work later.

File: source/sdk/flutter/sync/manage-sync-session.txt [hunk 2]
```diff
@@ -124,3 +132,28 @@ You can access the current and previous ``ConnectionState`` from ``ConnectionSta
...
+
+When you call this method, the SDK forces all sync sessions to attempt to 
+reconnect immediately and resets any timers used for incremental backoff.
+
+.. important:: Cannot Reconnect Within Socket Read Timeout Duration
+   
+   Realm has an internal default socket read timeout of 2 minutes, where 
+   Realm will time out if a read operation does not receive any data 
+   within a 2-minute window. If you call ``App.Sync.reconnect()`` 
+   within that window, the Kotlin SDK does *not* attempt to reconnect.
```
Comment (dacharyc) [inline source/sdk/flutter/sync/manage-sync-session.txt:159]: Oops. Missed copy-pasta.

```suggestion
   within that window, the Flutter SDK does *not* attempt to reconnect.
```

**Review by dacharyc — APPROVED** [review]
Overall, LGTM - just one small Q and a copy pasta fix.

File: source/sdk/flutter/sync/manage-sync-session.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/sdk/flutter/sync/manage-sync-session.txt:139]: Yeah, that's a good point. Thank you!

**Pull request merged**